src/styles/utils/mixins.scss

```
@@ -0,0 +1,5 @@
+@mixin circle ($size-vertical, $size-horizontal) {
+  width: $size-horizontal;
+  height: $size-vertical;
+  border-radius: 50%;
+}
```

if it's a circle you don't need to have 2 params
```suggestion
@mixin circle ($size) {
  width: $size;
  height: $size;
  border-radius: 50%;
}
```